src/main/java/com/monumental/controllers/MonumentController.java

```diff
@@ -92,6 +128,35 @@ public Monument updateMonument(@PathVariable("id") Integer id, @RequestBody Upda
         return this.monumentService.updateMonument(id, newMonument);
     }
 
+    private static class ToggleIsActiveRequest {
```

Comment:
Do you think it's worth it to add a comment how about why we decided to use a `private static class` vs. creating a new Request class for it?

Reply:
Not really. It's a simple wrapper. I think we'll be seeing more of these anyway